versions/1.1/SPEC.md

```diff
@@ -665,149 +690,37 @@ String a = "beware"
 String b = "key"
 String c = "lookup"
 
+struct Words {
+  Int a
+  Int b
+  Int c
+}
+
 # What are the keys to this object?
```

Comment:
Do you mean: "# What are the keys to this **`Struct`**?"

Reply:
yep good catch